File: javascript-closure-angular-client/API/Client/VehicleID.js
goog.provide('API.Client.VehicleID');

/**
 * @record
 */
API.Client.VehicleID = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.VehicleID.prototype.licensePlate;

/**
 * @type {!API.Client.StateEnum}
 * @export
 */
API.Client.VehicleID.prototype.stateCode;

/**
 * Cargo Transport National Register, Registro Nacional de Transportador de Carga (ANTT)
 * @type {!string}
 * @export
 */
API.Client.VehicleID.prototype.rtnc;

File: async-scala-client/src/main/scala/io/swagger/client/model/VehicleID.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class VehicleID (
  licensePlate: String,
stateCode: StateEnum,
rtnc: Option[String]  // Cargo Transport National Register, Registro Nacional de Transportador de Carga (ANTT)
)
